test/e2e/specs/onboarding/signup__videopress-tailored.ts

```diff
@@ -37,6 +37,7 @@ describe( DataHelper.createSuiteTitle( 'Signup: Tailored VideoPress' ), () => {
 		} );
 
 		it( 'Click Get Started', async function () {
+			page.setDefaultTimeout( 60 * 1000 );
```

Comment:
Calling this will extend the timeout to 1 minute for all methods (that accept the timeout option). In our setup, `page` is also a global value, so changing its default values from a test scope can have unwanted side effects. Let's move the timeout to the `click` action below – it looks like it's where we need the longer wait.